manage.py

```diff
@@ -108,6 +108,12 @@ def seed():
     # Scrape articles for feed
     scrape_articles(Categorizer())
 
+    # Add items to user_item relationship
```

Comment:
Ok sounds good.

-- 
Alex Gerstein
gerstein.alex@gmail.com

On February 20, 2015 at 2:57:39 PM, Scott Gladstone (notifications@github.com) wrote:

In manage.py:

> @@ -108,6 +108,12 @@ def seed():
>      # Scrape articles for feed
>      scrape_articles(Categorizer())
>   
> +    # Add items to user_item relationship
Testing queries mostly. It doesn't cost much to fill up the table with one user and one set of feed items, so I see little reason not to do it.

—
Reply to this email directly or view it on GitHub.